packages/components/src/FormField/FormField.module.css

```diff
@@ -117,8 +117,8 @@
 .disabled {
   --field--placeholder-color: var(--color-disabled);
   --field--value-color: var(--color-disabled);
-  --field--background-color: var(--color-surface--background);
-  --field--border-color: var(--color-disabled--secondary);
+  --field--background-color: var(--color-disabled--secondary);
+  --field--border-color: var(--color-border);
```

Comment:
👏 THANK YOU 
![image](https://github.com/user-attachments/assets/4cf42b87-aff2-4d5f-8096-4042e1067556)